### ISSUE TITLE
Docker Quick Start not workin

### ISSUE THREAD

**Issue by arjunmltrh** [nishadhka/howtotrainyourcomputer#1]
Hi,

After I install docker quick started and docker toolbox and pasteed boot2docker.iso file in C:\Users\User.docker\machine\cache\boot2docker.iso, I opened Docker quick start and followed the instructions as mentioned in the homepage. However, I am receiving an error message. I am attaching a screenshot of the error message. Please let me know how to move forward.

![image](https://user-images.githubusercontent.com/43564464/45996335-80f7b400-c0b9-11e8-8819-1ca894707b37.png)

**Comment by nishadhka** [comment]
Please mention your computer 32 bit or 64 bit and Windows OS version, if it windows 10 a restart of the computer was solving permission and most network issues

**Comment by arjunmltrh** [comment]
My Windows OS is Windows 10 Home Single Language 64-bit with 4GB ram. I have shutdown the laptop and then again tried running it but it did not work. I have also enabled virtualization by going into the BIOS setup.

**Comment by nishadhka** [comment]
It seems the iso file is having some problem, actually the docker toolbox automatically download the iso file provided with internet connectivity and create a virtual instance(based on iso)  where the docker works, can you please remove the iso file from the location and start the docker quick start

**Comment by arjunmltrh** [comment]
From this location C:\Users\User.docker\machine\cache\boot2docker.iso? In fact when I went to this location to paste the boot2docker.iso file, there was already a file called boot2docker.iso.tmp921682575. Do you want me to delete boot2docker.iso.tmp921682575 and paste boot2docker.iso or leave the location empty?

**Comment by nishadhka** [comment]
Make the location emtpy and restart the docker quick start. In your previous attempt, the iso file is pasted in the location is in it? as per the instructions

**Comment by arjunmltrh** [comment]
Its not working. Whenever it asks to press any key, I press enter and the application closes.

**Comment by nishadhka** [comment]
Sorry for delay in recreating issue in my computer, actually there is a issue possibly related with recent(6 days before) changes in boot2docker.iso source. I got error as follows
![screenshot_2018-09-25_19-55-43](https://user-images.githubusercontent.com/3928997/46023301-f8513600-c101-11e8-85bd-15f91afa5a7e.png)
It is found that the link for the iso files is becomes obsolete, have to download the iso file from [here](https://github.com/boot2docker/boot2docker/releases/download/v18.06.1-ce/boot2docker.iso) in which version v18.06.1 has to be downloaded. This file is a 45MB and has to kept in the file path  C:\Users\\$USER\\.docker\machine\cache\boot2docker.iso, where $USER is your username in the windows. After doing this running Docker Quick start terminal make ready with docker terminal.
![screenshot_2018-09-25_20-47-27](https://user-images.githubusercontent.com/3928997/46024506-8f1ef200-c104-11e8-808f-f70f261d4add.png)

**Comment by arjunmltrh** [comment]
It is still showing the same message, "Unable to start VM: exit status 1......."

**Comment by nishadhka** [comment]
Please point out the steps in leading to the error, steps in my case is as follows.

1. Local copy of Docker toolbox from [here](https://docs.docker.com/toolbox/toolbox_install_windows/) for windows 64 bit.
1. Local copy of boot2docker.iso from [here](https://github.com/boot2docker/boot2docker/releases/download/v18.09.0-ce-beta1/boot2docker.iso), I used v18.09.0 as iso file and the docker toolbox was later downloading the version v18.06.0. May be using this can be alternative step. 
1. save the iso file in C:\Users\\$USER\\.docker\machine\cache\boot2docker.iso, where $USER is your username in the windows
2. Open the program Docker quick start, It will make a separate virtual machine to run the docker container.
The version used for iso file can be found [here](https://github.com/boot2docker/boot2docker/releases)

**Comment by arjunmltrh** [comment]
I have completed steps 1 to 3 and the problem arises at the fourth step when I run the Docker quick start. Should I uninstall docker toolbox and .iso file and reinstall it? Also, you have mentioned two sources for downloading the .iso file (one in step 2 and one in the end). Which one should I use? In case, I have to use the one in the end, which file should I install? The link you have shared shows multiple links for downloads.

**Comment by nishadhka** [comment]
Please do the following
1. Make sure you have no iso file in the folder C:\Users\\$USER\\.docker\machine\cache\, where $USER is your username in the windows
1. Download this [file](https://github.com/boot2docker/boot2docker/releases/download/v18.09.0-ce-beta1/boot2docker.iso), earlier mentioned two links are same origin and we need version v18.09.0, so download that file.
1. Keep the iso file in folder C:\Users\\$USER\\.docker\machine\cache\
1. Start the Docker quick start, it took several minutes to complete, if it trying to download the iso file by its own, it shows different behaviour in my case as well, try our luck
 
Edit: the markdown for any path has to be double \\ for \\$ and \\.

**Comment by arjunmltrh** [comment]
Hey,

I tried that but it was not working. I have also tried to check "VirtualBox NDIS6 Bridged Networking driver" in the Virtualbox host only adapter (image below) that was created but it gave me a new error:

![image](https://user-images.githubusercontent.com/43564464/46066457-4a419c80-c192-11e8-9c77-28d4fd828d03.png)

![image](https://user-images.githubusercontent.com/43564464/46066598-87a62a00-c192-11e8-8cde-6dd5467feb3e.png)
 
In fact, I tried this multiple times but each time it kept giving me the above error. It has now created 8 Virtualbox host only adapters in my network and sharing centre.

![image](https://user-images.githubusercontent.com/43564464/46066796-eff50b80-c192-11e8-81dc-057277cf479c.png)

Should I uninstall and reinstall every file?

**Comment by nishadhka** [comment]
Yes, I guess, have to remove all the trace like these ethernets etc, it seems the iso file link has issue in all the cases.

**Comment by arjunmltrh** [comment]
Hi Nishadh,

Thanks for all the help you have provided. I uninstalled all the files and reinstalled them. I was able to run the docker after that! 

However when I ran the "docker exec -it container_ID bash" command, I got an error saying "No such container: container_ID". I am attaching a screenshot below:

![image](https://user-images.githubusercontent.com/43564464/46129642-f77ce900-c254-11e8-94ea-67b9dc80523e.png)

Please let me know how to proceed.

**Comment by nishadhka** [comment]
Hi, that is good news, regarding the container_ID, it is a variable, you have to enter the value it returns from ```docker ps```, from the screenshot your container_ID is  ```07f19```, that will proceed you to the terminal of image downloaded from dockerhub.
